vignettes/performance_analysis.Rmd

```diff
@@ -0,0 +1,323 @@
+---
+title: "Conducting Performance Analyses"
+output: rmarkdown::html_document
+vignette: >
+  %\VignetteIndexEntry{Conducting Performance Analyses}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+This vignette introduces the concept of a performance analysis and demonstrates how `eiCompare` can be used to conduct one given voter files and districting maps.
+
+```{r imports, echo=FALSE, message=FALSE, warning=FALSE}
+suppressMessages({
+  library(dplyr)
+  library(eiCompare)
+  library(ggplot2)
+  library(sf)
+  library(wru)
+})
+```
+
+## What is a performance analysis? 
+
+A successful voting rights case may result in a local jurisdiction's districting map to be thrown out, prompting the need for a new map. Thus, it's necessary assess whether the new map provides sufficient representation for minority groups. It’s not desirable to simply wait for elections to happen and see what the results might be. Instead, we can look at past elections, and observe how candidates would perform if the proposed map was used at the time of the election. This is the basis of a performance analysis.
```

Comment:
A successful voting rights case may result in a local jurisdiction's districting map being thrown out